Examples/Apply_scipy_function_to_every_gridpoint.ipynb

```diff
@@ -0,0 +1,533 @@
+{
```

Comment:
<p>This is great! Maybe a bit more text to help out a beginner? Eg what is vectorising / why do we care.</p><p> </p><p>"Imagine you want to apply this linear regression to every point in space. Looping over every x and y would do the trick but is <em>very slow</em>. Instead, a vectorised function avoids some of the python slowness by employing the fast numpy backend. The <code>xr.apply_ufunc</code> function lets us take a function, and apply it over our whole domain. ... " </p><p> </p><p>I'm tired so that's probably not written very well but you get the gist </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/421/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Yes. I like this suggestion. Perhaps have some pseudo code how would you do it with nested for loops and then say that this would be detrimental for performance so we want apply_ufunc?